.github/workflows/maven.yml

```diff
@@ -30,4 +30,4 @@ jobs:
         distribution: 'temurin'
         java-version: '11'
     - name: Build with Maven
-      run: mvn -B verify -Pdev --file pom.xml
+      run: mvn -B -o --file pom.xml -Dmaven.javadoc.skip=true verify
```

Comment:
Shouldn't we leave one platform with javadoc at least? Or is that being built on Jenkins?

Reply:
The release has to use Java17 outputting Java11 byte code because of Javadoc probems. Fixes have not been fully backported to Java11.

That means one check is needed - does the code compile on Java11 - not the javadoc.

It' probably less of an issue these days but it used to be possible to have JDK library calls at the level of the compiler, not the target, and then they weren't available at runtime.

This job checks that Java11 can compile the code.

Jenkins does make Javadoc, as do the other github actions (Linux, MacOS). It's this one case (Windows+Java11) that seems more unstable.

Reply:
>Jenkins does make Javadoc, as do the other github actions (Linux, MacOS). It's this one case (Windows+Java11) that seems more unstable.

:+1: thanks!